_episodes/16-verifying-code-style-linters.md

```diff
@@ -48,7 +48,7 @@ $ python3 -m pip freeze > requirements.txt
 Pylint is a command-line tool that can help our code in many ways:
 
 - **Check PEP8 compliance:**
-  whilst in-IDE context-sensitive highlighting such as that provided via PyCharm
+  whilst in-IDE context-sensitive highlighting such as that provided via VS Code
```

Comment:
Note that VS Code doesn't do the same context-sensitive highlight that PyCharm does - so we will need to reword, or delete, this sentence.

Reply:
Agreed, as above. We can check this.

Reply:
@douglowe, now that we have added the Flake8 extension in the previous episode, this sentence makes sense, but I'm adjusting it slightly to mention that we did the install of Flake8.